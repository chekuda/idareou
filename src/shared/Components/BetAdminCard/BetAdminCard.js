import React from 'react'
import Proptypes from 'prop-types'

if (process.browser) {
  require('./BetAdminCard.scss')
}

const BetAdminCard = ({
  title = '',
  description = '',
  amount = '',
  currency = ''
}) => {
  return <div className="betadmincard">
    <div className="betadmincard__section betadmincard__title">{title}</div>
    <div className="betadmincard__section betadmincard__description">{description}</div>
    <div className="betadmincard__section betadmincard__amount">{currency + amount}</div>
  </div>
}

BetAdminCard.propTypes = {
  title: Proptypes.string,
  description: Proptypes.string,
  amount: Proptypes.number,
  currency: Proptypes.string,
  options: Proptypes.array
}

export default BetAdminCard
