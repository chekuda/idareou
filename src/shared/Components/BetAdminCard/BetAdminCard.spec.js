import React from 'react'
import { shallow } from 'enzyme'

import BetAdminCard from './BetAdminCard'

describe('BetAdminCard', () => {
  it('renders without errors', () => {
    const component = shallow(<BetAdminCard />)

    expect(component).toMatchSnapshot()
  })
})
