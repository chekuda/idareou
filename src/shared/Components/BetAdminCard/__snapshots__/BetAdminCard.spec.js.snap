// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BetAdminCard renders without errors 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <BetAdminCard />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="betadmincard__section betadmincard__title"
        >
          
        </div>,
        <div
          className="betadmincard__section betadmincard__description"
        >
          
        </div>,
        <div
          className="betadmincard__section betadmincard__amount"
        >
          
        </div>,
      ],
      "className": "betadmincard",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "",
          "className": "betadmincard__section betadmincard__title",
        },
        "ref": null,
        "rendered": "",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "",
          "className": "betadmincard__section betadmincard__description",
        },
        "ref": null,
        "rendered": "",
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "",
          "className": "betadmincard__section betadmincard__amount",
        },
        "ref": null,
        "rendered": "",
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="betadmincard__section betadmincard__title"
          >
            
          </div>,
          <div
            className="betadmincard__section betadmincard__description"
          >
            
          </div>,
          <div
            className="betadmincard__section betadmincard__amount"
          >
            
          </div>,
        ],
        "className": "betadmincard",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "",
            "className": "betadmincard__section betadmincard__title",
          },
          "ref": null,
          "rendered": "",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "",
            "className": "betadmincard__section betadmincard__description",
          },
          "ref": null,
          "rendered": "",
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "",
            "className": "betadmincard__section betadmincard__amount",
          },
          "ref": null,
          "rendered": "",
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
